### PR TITLE
[skip ci] Update test case.

### PR DESCRIPTION
**Pull request by binbinlv**
Update test case.

Signed-off-by: Binbin Lv <binbin.lv@zilliz.com>

### CODE REVIEW

File: tests20/python_client/base/collection_wrapper.py [hunk 1]
```diff
@@ -135,6 +135,13 @@ def drop_partition(self, partition_name, check_task=None, check_items=None, **kw
         check_result = ResponseChecker(res, func_name, check_task, check_items, check, partition_name=partition_name, **kwargs).run()
         return res, check_result
 
+    def create_partition(self, partition_name, check_res=None, check_params=None, **kwargs):
```
Comment (yanliang567) [inline tests20/python_client/base/collection_wrapper.py:138]: check_params -> check_items

File: tests20/python_client/testcases/test_search.py [hunk 2]
```diff
@@ -135,559 +134,636 @@ def clear_env(self):
             assert not ("search_collection" in res)
         log.info("clear_env: cleared env (deleted %s search collections)" % count)
 
+    def search_results_check(self, search_res, nq=default_nq, limit=default_limit):
```
Comment (yanliang567) [inline tests20/python_client/testcases/test_search.py:137]: can we make this method as a check task in class ResponseChecker

File: tests20/python_client/base/collection_wrapper.py (hunk 1 above)
Comment (binbinlv) [reply tests20/python_client/base/collection_wrapper.py:138]: OK

File: tests20/python_client/testcases/test_search.py (hunk 2 above)
Comment (binbinlv) [reply tests20/python_client/testcases/test_search.py:137]: OK

File: tests20/python_client/check/func_check.py [hunk 3]
```diff
@@ -117,3 +120,35 @@ def check_partition_property(partition, func_name, check_items):
         if check_items.get("num_entities", None):
             assert partition.num_entities == check_items["num_entities"]
         return True
+<<<<<<< Updated upstream
+=======
```
Comment (FluorineDog) [inline tests20/python_client/check/func_check.py:124]: conflicts?

**Review by FluorineDog — CHANGES_REQUESTED** [review]
this pr has conflicts in the code

File: tests20/python_client/check/func_check.py (hunk 3 above)
Comment (binbinlv) [reply tests20/python_client/check/func_check.py:124]: OK, thanks.

**Review by FluorineDog — APPROVED** [review]

**Review by yanliang567 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**